### PR TITLE
[WFCORE-6016] Modify description in remoting of subsystem to its corr…

### PR DESCRIPTION
**Pull request by moulalis**
Issue: https://issues.redhat.com/browse/WFCORE-6016

### CODE REVIEW

File: remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties [hunk 1]
```diff
@@ -26,12 +26,12 @@ remoting.max-outbound-messages=The maximum number of concurrent outbound message
 remoting.send-buffer-size=The size of the largest buffer that this endpoint will transmit over a connection.
 remoting.max-inbound-messages=The maximum number of concurrent inbound messages on a channel.
 remoting.receive-window-size=The maximum window size of the receive direction for connection channels, in bytes.
-remoting.heartbeat-interval=The interval to use for connection heartbeat, in milliseconds.  If the connection is idle in the outbound direction \
-  for this amount of time, a ping message will be sent, which will trigger a corresponding reply message.
+remoting.heartbeat-interval=The interval to use for the connection heartbeat, in milliseconds. If the connection is idle in the outbound direction  \
+  for this amount of time, a ping message will be sent, which will trigger a corresponding reply message..
```
Comment (yersan) [inline remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties:30]: There are double dots at the end.

Comment (moulalis) [reply remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties]: Changes updated.
Thanks.

File: remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties [hunk 2]
```diff
@@ -7,13 +7,13 @@ remoting.worker-task-core-threads=The number of core threads for the remoting wo
 remoting.worker-task-core-threads.deprecated=Use IO subsystem worker configuration
 remoting.worker-task-keepalive=The number of milliseconds to keep non-core remoting worker task threads alive.
 remoting.worker-task-keepalive.deprecated=Use IO subsystem worker configuration
-remoting.worker-task-limit=The maximum number of remoting worker tasks to allow before rejecting.
+remoting.worker-task-limit=The maximum number of remote worker tasks to allow before rejecting them.
```
Comment (bstansberry) [inline remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties:10]: Unless @fl4via disagrees, I think 'remoting' is correct. It's not an adjective describing the compound noun 'worker tasks', such that the tasks are 'remote' (from what?). It's a 3rd noun in a compound noun, 'remoting work tasks', with 'remoting' meaning JBoss Remoting.

Perhaps it should be more clear and say 'JBoss Remoting' or perhaps 'remoting' should just be dropped.

File: remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties [hunk 3]
```diff
@@ -112,7 +112,7 @@ remote-outbound-connection.authentication-context=Reference to the authenticatio
 remote-outbound-connection.property=The XNIO Options that will be used during the connection creation.
 
 
-remoting.local-outbound-connection=Remoting outbound connections for local:// URI scheme.
+remoting.local-outbound-connection=Remotely establishing outbound connections for the local://URI scheme.The remoting connectors.
```
Comment (bstansberry) [inline remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties:115]: Similar issue as my comment above. Also, the space between local:// and URI should be restored. The string 'URI' is not part of the URI, it's a separate word.

**Comment by moulalis** [comment]
@fl4via Could you lease review it?
Thanks

**Pull request closed without merging**